packages/mobile/src/backup/BackupQuiz.tsx

```diff
@@ -82,21 +84,32 @@ export class BackupQuiz extends React.Component<Props, State> {
     this.setBackSpace()
   }
 
-  componentDidMount() {
-    const mnemonic = this.getMnemonicFromNavProps()
+  componentDidMount = async () => {
+    const mnemonic = await this.retrieveMnemonic()
     const shuffledMnemonic = getShuffledWordSet(mnemonic)
```

Comment:
Should you check for null/empty here since that's what the method returns on error

Reply:
hmm a good question. I wasn't sure exactly how to handle the empty case. right now retrieveMnemonic returns an empty string if there is an error.

I guess it probably is better to short circuit if there isnt a mneumonic aye?

Reply:
ok looking that the code more it seems very very very unlikely mnemonic will ever be blank. we already Log and throw errors in several places when it is. I don't think there is much to be gained from doing an empty string check here.